packages/pluggableWidgets/slider-native/src/__tests__/Slider.spec.tsx

```diff
@@ -32,21 +32,51 @@ describe("Slider", () => {
         const component = render(
             <Slider {...defaultProps} valueAttribute={new EditableValueBuilder<Big>().isLoading().build()} />
         );
-        expect(component.UNSAFE_queryByType(Text)).toBeNull();
+        expect(component.queryByTestId(`${defaultProps.name}-validation-message`)).toBeNull();
     });
 
     it("renders an error when the minimum is greater than the maximum", () => {
         const component = render(<Slider {...defaultProps} minimumValue={dynamicValue(new Big(300))} />);
-        expect(component.UNSAFE_getByType(Text).props.children).toBe(
-            "The minimum value can not be greater than the maximum value."
-        );
+        expect(component.queryByText("The minimum value can not be greater than the maximum value.")).not.toBeNull();
     });
 
     it("renders an error when the step size is negative", () => {
         const component = render(<Slider {...defaultProps} stepSize={dynamicValue(new Big(-10))} />);
-        expect(component.UNSAFE_getByType(Text).props.children).toBe(
-            "The step size can not be zero or less than zero."
+        expect(component.queryByText("The step size can not be zero or less than zero.")).not.toBeNull();
+    });
+
+    it("renders an error when the step size is empty", () => {
+        const component = render(
+            <Slider {...defaultProps} stepSize={{ status: ValueStatus.Unavailable, value: undefined }} />
         );
+        expect(component.queryByText("No step size provided.")).not.toBeNull();
+    });
+
+    it("renders an error when the minimum is empty", () => {
+        const component = render(
+            <Slider {...defaultProps} minimumValue={{ status: ValueStatus.Unavailable, value: undefined }} />
+        );
+        expect(component.queryByText("No minimum value provided.")).not.toBeNull();
+    });
+
+    it("renders an error when the maximum is empty", () => {
+        const component = render(
+            <Slider {...defaultProps} maximumValue={{ status: ValueStatus.Unavailable, value: undefined }} />
```

Comment:
This `{ status: ValueStatus.Unavailable, value: undefined }` is used often, do you want to make it a var?

Reply:
Done